blueman/main/NetworkManager.py

```diff
@@ -19,9 +19,9 @@ class NMConnectionError(Exception):
 class NMConnectionBase:
     conntype: str
 
-    def __init__(self, service, reply_handler=None, error_handler=None):
+    def __init__(self, service, reply_handler, error_handler):
         if self.conntype not in ('dun', 'panu'):
-            self._raise_or_error_handler(
```

Comment:
This was actually wrong as it'd raise an exception in the async call instead of calling the error handler.